pkgs/development/python-modules/pdf2image/default.nix

```diff
@@ -2,13 +2,13 @@
 
 buildPythonPackage rec {
   pname = "pdf2image";
-  version = "1.0.0";
+  version = "1.3.1";
 
   buildInputs = [ pillow poppler_utils ];
```

Comment:
`pillow` should be a `propagatedBuildInputs` https://github.com/Belval/pdf2image/blob/8cf6e7773359521c07a82cee786532d918273b2e/setup.py#L48

Not sure about `poppler_utils` though.